src/ios/CDVCapture.m

```diff
@@ -292,6 +295,65 @@ - (CDVPluginResult*)processVideo:(NSString*)moviePath forCallbackId:(NSString*)c
     return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:fileArray];
 }
 
+- (void)showAlertIfAccessProhibeted
+{
```

Comment:
For styling consistency, can we keep the opening brace of a function on the same line as the function?